### PR TITLE
finishAuthTests

### PR DESCRIPTION
**Pull request by Avazbek22**

### CODE REVIEW

File: tests/MRA.Jobs.Identity/Application.IntegrationTests/RegistrationTest.cs [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        {
+            Email = "test@example.com",
+            Password = "password@#12P",
+            FirstName = "Alex",
+            Username = "@Alex22",
+            LastName = "Makedonskiy",
+            PhoneNumber = "123456789"
+        };
+        
+        var content = new StringContent(JsonConvert.SerializeObject(request), Encoding.UTF8, "application/json");
```
Comment (komdil) [inline tests/MRA.Jobs.Identity/Application.IntegrationTests/RegistrationTest.cs:32]: use PostAsJsonAsync

File: tests/MRA.Jobs.Identity/Application.IntegrationTests/RegistrationTest.cs [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            FirstName = "Alex",
+            Username = "@Alex22",
+            LastName = "Makedonskiy",
+            PhoneNumber = "123456789"
+        };
+        var content = new StringContent(JsonConvert.SerializeObject(request), Encoding.UTF8, "application/json");
+
+        // Act
+        await _client.PostAsync("api/Auth/register", content);
+        var result = _context.Users.Count();
```
Comment (komdil) [inline tests/MRA.Jobs.Identity/Application.IntegrationTests/RegistrationTest.cs:58]: This should be under assert comment

File: tests/MRA.Jobs.Identity/Application.IntegrationTests/RegistrationTest.cs [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+            Password = "password", // incorrect password
+            FirstName = "Alex",
+            Username = "@Alex22",
+            LastName = "Makedonskiy", 
+            PhoneNumber = "123456789"
+        };
+        var content = new StringContent(JsonConvert.SerializeObject(request), Encoding.UTF8, "application/json");
+    
+        // Act
+        var response = await _client.PostAsync("api/Auth/register", content);
```
Comment (komdil) [inline tests/MRA.Jobs.Identity/Application.IntegrationTests/RegistrationTest.cs:80]: use PostAsJsonAsync

**Review by komdil — CHANGES_REQUESTED** [review]

**Review by komdil — DISMISSED** [review]

File: tests/MRA.Jobs.Identity/Application.IntegrationTests/RegistrationTest.cs [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+            PhoneNumber = "123456789"
+        };
+
+        // Act
+        var response = await _client.PostAsJsonAsync("api/Auth/register", request);
+
+        // Assert
+        Assert.That(response.StatusCode, Is.EqualTo(HttpStatusCode.OK));
+    }
+
```
Comment (Firuz2006) [inline tests/MRA.Jobs.Identity/Application.IntegrationTests/RegistrationTest.cs:32]: I recommend removing this test

File: tests/MRA.Jobs.Identity/Application.IntegrationTests/RegistrationTest.cs [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+            Password = "password@#12P",
+            FirstName = "Alex",
+            Username = "@Alex22",
+            LastName = "Makedonsky",
+            PhoneNumber = "123456789"
+        };
+        
+        // Assert
+        var response = await _client.PostAsJsonAsync("api/Auth/register", request);
+       // Assert.That(_context.Users.Count(), Is.EqualTo(1));
```
Comment (Firuz2006) [inline tests/MRA.Jobs.Identity/Application.IntegrationTests/RegistrationTest.cs:49]: enable database checking

**Review by Firuz2006 — CHANGES_REQUESTED** [review]

**Review by Firuz2006 — APPROVED** [review]

**Review by komdil — APPROVED** [review]

**Pull request merged**